### ISSUE TITLE
https://v2.vuepress.vuejs.org/ stuck at beta.26

### ISSUE THREAD

**Issue by JohannesRudolph** [vuepress/vuepress-next#643]
Just wanted to make you aware that the official documentation site https://v2.vuepress.vuejs.org/ seems to be stuck at beta.26 
![image](https://user-images.githubusercontent.com/130103/150309600-ad9deabd-95eb-4a8e-93a4-4fdf95107bb9.png)

Noticed this while trying to update to latest beta and noticed that breaking changes mentioned in the release notes were not reflected on the documentation site yet.

**Comment by meteorlxy** [comment]
Really sorry for that.

We removed PWA from our doc site, because the PWA update process took quite a long time, and we did not see much benifit of PWA.

So the version would get stuck if users had visited our site previously.

To fix that, press `F12` to open devtools, nav to `Application` tab, and manually remove the service worker:

![image](https://user-images.githubusercontent.com/18205362/150313782-5970fb43-f812-4e28-bfb6-e34efda6d2d6.png)

Apologize again
